guides/06_FAQs.md

```diff
@@ -0,0 +1,239 @@
+
+Amazon Chime SDK for iOS https://github.com/aws/amazon-chime-sdk-ios
+Amazon Chime SDK for Android https://github.com/aws/amazon-chime-sdk-android
+
+## Meetings
+
+### How do users authenticate into a meeting?
+
+Amazon Chime SDK uses join tokens to participate in Amazon Chime meetings that are obtained when each attendee is [created](https://docs.aws.amazon.com/chime/latest/dg/mtgs-sdk-mtgs.html). The safe distribution of these tokens to the clients is the application developer’s responsibility. Amazon Chime SDK does not authenticate end users. Developers could use services such as [Amazon Cognito](https://aws.amazon.com/cognito/), Open ID connect, Active Directory, or Facebook to log in end users to their application. You can tie external identity management with Amazon Chime SDK attendees using externalUserId argument supplied at the attendee creation time.
+
+### When does an Amazon Chime SDK meeting end?
+
+An Amazon Chime SDK meeting ends when you run the [DeleteMeeting](https://docs.aws.amazon.com/chime/latest/APIReference/API_DeleteMeeting.html) API action. Also, a meeting automatically ends after a period of inactivity, based on the following rules:
+
+* No audio connections are present in the meeting for more than five minutes.
+* Only one audio connection is present in the meeting for more than 30 minutes.
+* 24 hours have elapsed since the meeting was created.
+
+### How many simultaneous meetings can be hosted in an account? Can this limit be raised?
+
+AWS accounts have a soft limit of [250 concurrent meetings](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-limits). Your limit can be obtained from the [Service quotas console](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) in your AWS account. The limit is applied globally, based on the total number of active meetings across all regions. You can track your concurrent meeting usage in [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/).  If your application needs to support more concurrent meetings, you can request limit increases through the [AWS Customer support portal](https://pages.awscloud.com/GLOBAL-aware-GC-Amazon-Chime-SDK-2020-reg.html).
+
+### How many attendees can join an Amazon Chime SDK meeting? Can this limit be raised?
+
+Amazon Chime SDK limits are defined [here](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-limits). The service supports up to 250 attendees and up to 16 video participants in a meeting (video limit is enforced separately). An attendee is considered active unless it has been explicitly removed using DeleteAttendee API call. Attendee limits cannot be changed. 
+
+If your use case requires more than 250 attendees, consider using a [broadcasting solution](https://quip-amazon.com/e8N3AnXbdH1w/Chime-Javascript-SDK-Frequently-Asked-Questions-DRAFT#UDQ9CAA1oPv).
```

Comment:
Replace the Quip doc with the public GitHub URL

Reply:
good one